updated readme (#78)
Altinn/altinn-notifications-email: README.md

## Getting started

1. [.NET 8.0 SDK](https://dotnet.microsoft.com/download/dotnet/8.0)
2. Newest [Git](https://git-scm.com/downloads)
3. A code editor - we like [Visual Studio Code](https://code.visualstudio.com/download)
   - Also install [recommended extensions](https://code.visualstudio.com/docs/editor/extension-marketplace#_workspace-recommended-extensions) (e.g. [C#](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp))
4. [Podman](https://podman.io/) or another container tool such as Docker Desktop

### Cloning the application

Clone [Altinn Notifications Email repo](https://github.com/Altinn/altinn-notifications-email) and navigate to the folder.

```bash
git clone https://github.com/Altinn/altinn-notifications-email
cd altinn-notifications-email
```

### Setting up Kafka broker and visualization

Ensure that Docker and Podman are installed and is running.

In a terminal navigate to the root of this repository
and run command `podman compose -f setup-kafka.yml up -d`

> Kafdrop is not available at http://localhost:9000.

Author: ```suggestion
Kafdrop is now available at http://localhost:9000.
```

### Set up Azure Communication Services

If you need working end to end functionality when working on

> Notifications Email Azure Communication Services (ACS) needs to be set up. *

Author: ```suggestion
Notifications Email Azure Communication Services (ACS) needs to be set up.
```

Set up a service in your personal Azure account or use an existing service in a test environment.
Find the connection string in the Azure Portal under _Settings_ -> _Keys_ and add this to the configuration values.

We recommend settings it up as a user secret with the commands below.

```
cd src Altinn.Notifications.Email
dotnet user-secrets init
dotnet user-secrets set "CommunicationServicesSettings:ConnectionString" "insert-connection-string"
```

### Running the application with .NET

> The Notifications components can be run locally when developing/debugging. Follow the install steps above if this has not already been done.

Author: ```suggestion
The Notifications Email component can be run locally when developing/debugging. Follow the install steps above if this has not already been done.
```
